src/command.c

```diff
@@ -453,6 +456,60 @@ int cmd_proxy(struct command *cmd, struct redis_data *data)
 
     if (strcasecmp(type, "INFO") == 0) {
         return cmd_proxy_info(cmd);
+    } else if (strcasecmp(type, "SETREMOTES") == 0) {
```

Comment:
sure to change this?
because the PR in redisctl almost be merged so we must decide it quickly.

CONFIG SET NODE HOST1 PORT1 HOST2 PORT2 ... ?